styles/styles.css

```diff
@@ -483,3 +483,12 @@ main .section.heading-light h2 {
     line-height: 2.25rem;
   }
 }
+
+/* Underwriter page */
+main .underwriter h1 {
+  text-align: left;
+}
+
+main .underwriter h1 > span {
```

Comment:
Why not just `h1 > span`? It looks more clear that Icon in title should be spaced.